packages/icons-react/src/icons/pen/PenMedium.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { FC, SVGAttributes } from "react";
+
+export const PenMedium: FC<SVGAttributes<SVGElement>> = (props) => (
+    <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 16 16" {...props}>
```

Comment:
Er vi sikker om viewBox verdier her? Jeg har sammelignet med de andre tilsvarende filer og de bruker det sånn:

viewBox="0 0 24 24"

Reply:
Hvis du tenker på en SVG som et evig stort koordinatsystem så bestemmer viewBox hvilken del av det som skal være synlig. Dersom de som tegnet ikonet holdt seg mellom (0,0), (0,16), (16, 0) og (16,16) koordinatene så er det den viewBox'en vi vil ha. Hvis vi endrer til `0 0 24 24` her så vil vi få et tomt hvitt felt nederst til høyre for ikonet siden det vil bli med et område der det ikke er tegnet noe.

På https://jokul.fremtind.no/preview/3888-pen-icon/komponenter/icons/ vil det se slik ut (dersom man redigerer viewBox i DOM inspectoren)
![image](https://github.com/fremtind/jokul/assets/379849/91277cb9-33a1-426a-8c03-3184e3f3abaa)

Reply:
Takk for forklaring :)